RELEASE_NOTES.rst

```diff
@@ -1,3 +1,133 @@
+release to the coala gitter channel, blog, and twitter. We encourage all users
+interested in checking out new features or helping to test these pre-releases
+on their projects. If you do find a regression in a release candidate, please
+report it to the issue tracker with the word ``Regression`` somewhere in the
+title.
+
+**Breaking Changes**
+
+- coala exit codes have changed. If you relied on coala in scripts or
+  other automated systems please check if it it still works with the new
+  behavior. You can find the new exit codes
+  `here <http://docs.coala.io/en/latest/Users/Exit_Codes.html>`__.
+  `PR #3905 <https://github.com/coala/coala/pull/3905>`_
+
+**General**
+
+- Section Inheritance had been added to the coafile parsing. With this
+  change we are deprecating implicit ``default`` section inheritance.
+  Instead, you can now have definite section inheritance by using the
+  ``.`` in the section title. Append operators are now supported allowing
+  addition of values to inherited section. See the ``Mockup`` section of
+  `cEP-0005 <https://github.com/coala/cEPs/blob/master/cEP-0005.md>`_
+  for a syntax example and `PR #3178 <https://github.com/coala/coala/pull/3178/>`_
+  for more information.
+- The section name ``cli`` is reserved for cli arguments and should not be
+  used in the coala configuration file (``coafile``). If you are currently
+  using it in a configuration file, coala will warn you.
+- coala now outputs absolute files paths when a bear fails. This should
+  make debugging bear failures easier when using custom bears.
+
+**Usability**
+
+- A new placeholder, ``source_lines``,  has been added to the coala
+  ``--format`` option. Use it to output the affected lines in the source file
+  when a result occurs.
+- Bears are displayed alphabetically when listed using ``--show-bears``.
+- coala outputs a warning if a setting is overridden in the same section
+  since this is probably not what the user intended.
+- ``--format`` takes precedence over ``--ci`` when both arguments are issued
+- coala will report the version of the dependency required when the version
+  check fails to help the user install the correct version.
+  `Issue #3982 <https://github.com/coala/coala/issues/3982>`_
+
+**Developers**
+
+- ``@Linter`` decorator works with global bears. See our bear writing
+  `docs <http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html#global-bears>`__
+  for more information on how to write a global bear with the ``@Linter``
+  decorator.
+- ``Bear.DownloadCachedFile`` raises exceptions when the download fails.
+  Please handle this exception if the file download has a good chance
+  of failing.
+- Initial Next Generation Bear code has been merged and can be seen in
+  ``coalib/core``. Please do not write bears for this API yet as it has
+  not been integrated with the coala interface yet.
+- The ``deprecate_settings`` decorator now supports multiple deprecations
+  of the same setting.
```

Comment:
I think a link to some details would be good, imo this is not directly understandable ;)